src/components/Thanks/ThanksPageShare.vue

```diff
@@ -2,7 +2,6 @@
 	<div>
 		<ShareStepper
 			:lender-name="lender.firstName"
-			:show-lender-name="showLenderName"
```

Comment:
Do we want to remove this prop from `ShareStepper`?

Reply:
No, I want to use the shareStepper for a guest version, where I will use this prop as `false`